python/example_code/iam/tag_role.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2010-2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+
+import boto3
+
+
+# Create IAM client
+iam = boto3.client('iam')
+
+role_name='AccountantRole1'
+
+tags=[
+    {
+        'Key': 'Department',
+        'Value': 'Accounting'
+    },
+    {
+        'Key': 'Environment',
+        'Value': 'Production'
+    }
+]
+
+try:
+    response = iam.tag_role(
+        Tags=tags,
+        RoleName=role_name
+    )
+    print(response)
+except Exception as e:
+    print(e)
+
+#snippet-comment:[These are tags for the AWS doc team's sample catalog. Do not remove.]
+#snippet-sourcedescription:[attach_role_policy.py demonstrates how to Tag an IAM Role.]

```

Comment:
the file name needs updated 😸

Reply:
Ah, sorry about that, so much metadata now!